frontend/src/components/sidebar/SideBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Drawer from "react-modern-drawer";
+import "react-modern-drawer/dist/index.css";
+import { NavLink } from "react-router-dom";
+import classes from "./SideBar.module.scss";
+import { ReactComponent as SettingsIcon } from "../../assets/icons/settings-circle.svg";
+import { ReactComponent as ConnectIcon } from "../../assets/icons/connect-circle .svg";
+import { ReactComponent as SwitchIcon } from "../../assets/icons/switch.svg";
+import { ReactComponent as PlusIcon } from "../../assets/icons/plus.svg";
+import IconButton from "components/icon-button/IconButton";
+
+export type Sidebar = {
+  toggled: boolean;
+  toggleSidebar: () => void;
+};
+
+function SideBar({ toggled, toggleSidebar }: Sidebar) {
+  // TODO(sideBar-component): Check if it is possible (with currently used library "react-modern-drawer")
+  // to position sidebar bellow navbar instead of overlaping it
+  return (
+    <Drawer
+      open={toggled}
+      direction={"right"}
+      onClose={toggleSidebar}
+      className={classes.sidebar}
+      size={300}
+    >
+      <div className={classes.menu}>
+        <div>
+          <NavLink to={"#"} className={classes.menuItem}>
+            <SwitchIcon className={classes.icon} />
+            <div className={classes.text}>Switch project</div>
+          </NavLink>
+
+          <NavLink to={"#"} className={classes.menuItem}>
+            <ConnectIcon className={classes.icon} />
+            <div className={classes.text}>Connect</div>
+          </NavLink>
+
+          <NavLink to={"#"} className={classes.menuItem}>
+            <SettingsIcon className={classes.icon} />
+            <div className={classes.text}>Settings</div>
+          </NavLink>
+        </div>
+        <div className={`${classes.menuItem} ${classes.footer}`}>
+          <IconButton
+            icon={<PlusIcon></PlusIcon>}
+            iconPosition="before"
+            className={classes.button}
+          >
+            NEW PROJECT
+          </IconButton>
+          <p>TODO: Show emulator connection status</p>
+        </div>
+      </div>
+    </Drawer>
+  );
+}
+
+export default SideBar;
```

Comment:
Here I'd also prefer that we use "named export" instead of default export.

See the comment I made in the previous PR: https://github.com/onflowser/flowser/pull/94#discussion_r958979667

It would be nice that we'd enforce that kind of rules at pre-commit time instead of addressing them in PRs. There is an eslint plugin/rule that does just that, so maybe we could enable that sometime in the future: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-default-export.md